core/src/main/scala/pureconfig/backend/ConfigFactoryWrapper.scala

```diff
@@ -22,15 +22,17 @@ object ConfigFactoryWrapper {
     unsafeToEither(ConfigFactory.load())
 
   /** @see `com.typesafe.config.ConfigFactory.parseFile()` */
-  def parseFile(path: Path): Either[ConfigReaderFailures, Config] =
-    unsafeToEither(ConfigFactory.parseFile(path.toFile))
+  def parseFile(path: Path): Either[ConfigReaderFailures, Config] = {
+    if (!path.toFile.isFile || !path.toFile.canRead) Left(ConfigReaderFailures(CannotReadFile(path)))
```

Comment:
I wonder if it would be better to use the Java 7 NIO APIs in the conditional:

Files.isReadable(path) && Files.isRegularFile(path)